packages/node_modules/@ciscospark/widget-space-demo/src/components/demo-widget-space/index.js

```diff
@@ -229,8 +237,30 @@ class DemoWidgetSpace extends Component {
                 </RadioButtonGroup>
               </div>
 
+              <h3> Widget Initial Activity </h3>
+              <div className={classNames(styles.select)}>
+                <RadioButtonGroup
+                  aria-label="Initial Activity"
+                  name="initialActivity"
+                  onChange={this.handleInitialActivityChange}
+                  valueSelected={this.state.initialActivity}
+                >
+                  <RadioButton
+                    aria-label="Message"
+                    label="Message"
+                    value="message"
+                  />
+                  <RadioButton
+                    aria-label="Meet"
+                    label="Meet"
+                    value="meet"
+                  />
+                </RadioButtonGroup>
+              </div>
+
               <div>
                 <h3> Widget Activities </h3>
+                {displayActivityWarning && (<div> Warning: The initial activity must be selected </div>)}
```

Comment:
Does `Warning: The initial activity is not enabled` make more sense? When I first read this error, I thought it was that the initial activity wasn't chosen (selected).

Reply:
We should probably also put that string into a i18n constant.

Reply:
@adamweeks yeah that's probably a better way to phrase it and @bzang will do

Reply:
@bzang I don't believe we have any intl support in the demos

Reply:
Oops, sorry. I confused myself for a moment. Yea normal text for this is fine until we get the internationalization requirement.